tsconfig.json

```diff
@@ -1,7 +1,12 @@
 {
   "extends": "@tsconfig/node16/tsconfig.json",
   "compilerOptions": {
+    "lib": ["ES2022"],
```

Comment:
Why did you need to update this? The setting in `@tsconfig/node16` should be good.